README.md

```diff
@@ -25,3 +25,8 @@ $ git clone https://github.com/grailbio/bigslice
 $ cd bigslice
 $ GO111MODULE=on go test
 ```
+
+If tests fail with `socket: too many open files` errors, try increasing the maximum number of open files by running
```

Comment:
```suggestion
If tests fail with `socket: too many open files` errors, try increasing the maximum number of open files.
```

Reply:
FYI, you can directly commit "suggestions" by clicking the `Commit suggestion` button. This basically saves the work of copying over suggestions yourself. (It's one of my favorite features of GitHub's code reviews.)